### ISSUE TITLE
Request: Add the turret as a controllable entity.

### ISSUE THREAD

**Issue by Cornish-Kap** [ControlCompany/ControlCompany#1]
I would LOVE to be able to become a turret and sit in dumb places.

I imagine it like this:

I can move like any other enemy while being a turret, but it will only work as an actual turret once you hit a button to make it stationary (stuck in place) then you can aim it and shoot. Ideally the turret wind up will remain to give them a chance (optional).

**Comment by JustCallMeQuincy** [comment]
Yeah, spawning it from Ghost mode would plop you down as an immovable turret

LMB could be shooting and RMB could be pointing the laser

**Comment by JustCallMeQuincy** [comment]
A landmine could be added using a similar concept.
Once you become the landmine, you're stationary

LBM could be exploding and RMB could be "blipping", as if someone were close-by
